src/ZipFile.jl

```diff
@@ -32,6 +32,16 @@ Julia
 Julia
 Julia
 ```
+
+# Example 
```

Comment:
I don't see a reason to add another example. You can modify the example above to add a file inside a sub-directory.